srcpkgs/CalculiX-ccx/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'CalculiX-ccx'
+pkgname=CalculiX-ccx
+version=2.20
+revision=1
+wrksrc=CalculiX
+_ccx_dir=ccx_${version}
+_LIBS=" ${XBPS_CROSS_BASE}/usr/lib/spooles.a ${XBPS_CROSS_BASE}/usr/lib/libarpack.so ${XBPS_CROSS_BASE}/lib/libopenblas.so  -lpthread -lm -lc"
+hostmakedepends="perl gcc-fortran"
+makedepends="SPOOLES arpack-ng-devel libgomp-devel openblas-devel"
+_FFLAGS=" -Wall -O2 -fallow-argument-mismatch"
+
+if [ "$XBPS_MACHINE" == "x86_64" ]  && [ "$XBPS_TARGET_MACHINE" == "i686" ]; then
+	makedepends+=" libquadmath"
+	hostmakedepends+=" libgomp-devel-32bit"
+fi
+short_desc="3D Structural Finite Element Program - Solver part"
+maintainer="Phicem <phicem@gmx.com>"
+license="GPL-2.0-or-later"
+homepage="http://www.dhondt.de/"
+distfiles="http://www.dhondt.de/ccx_${version}.src.tar.bz2"
+checksum=63bf6ea09e7edcae93e0145b1bb0579ea7ae82e046f6075a27c8145b72761bcf
+CFLAGS=" -Wall -O2  -I ${XBPS_CROSS_BASE}/usr/include/SPOOLES-2.2 -DARCH='Linux' -DSPOOLES -DARPACK -DMATRIXSTORAGE -DNETWORKOUT"
+CFLAGS+=" ${XBPS_CROSS_CFLAGS}"
+_FFLAGS+=" ${XBPS_CROSS_CFLAGS}"
```

Comment:
```suggestion

FFLAGS="-fallow-argument-mismatch"
CFLAGS="-I${XBPS_CROSS_BASE}/usr/include/SPOOLES -DARCH=Linux -DSPOOLES -DARPACK -DMATRIXSTORAGE -DNETWORKOUT"
```
`xbps-src` integrates the CFLAGS/FFLAGS specified at the top level of the template into its own, and then exposes that to the packaging phases, so we can trim this down